config/overlays/tls/resources/manager_cert.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: olmv1-cert
+spec:
+  secretName: olmv1-cert
+  dnsNames:
+    - operator-controller.olmv1-system.svc
+    - operator-controller.olmv1-system.svc.cluster.local
+  privateKey:
+    algorithm: ECDSA
+    size: 256
+  issuerRef:
+    name: olmv1-ca
+    kind: ClusterIssuer
+    group: cert-manager.io
```

Comment:
This introduces a dependency on `ClusterIssuer` which is only being created In `install.tpl.sh`/in Tilt.

This means it is no longer possible to do this to install:

```
kubectl apply -f https://github.com/operator-framework/catalogd/releases/download/$CATALOGD_VERSION/catalogd.yaml
kubectl apply -f https://github.com/operator-framework/operator-controller/releases/download/$VERSION/operator-controller.yaml
```

You have to checkout the repo run the install script.

Are we ok with that?

Reply:
Also about related changes in `scripts/install.tpl.sh` where we create CA, etc.

As far as I understand - we need to create CA and patch deployment of the manger only for E2E setups. If it is the case - can we put this in `config/overlays/e2e/`?

I would like to avoid having to create all this on every cluster just because our E2E setup requires it.

Reply:
We need cert-manager regardless, AFAICT, as catalogd now creates certificates. So, the method listed above no longer works (or at least it shouldn't).
I tried to put this into the e2e overlay, but could not get the namespace to work as expected. Please see https://github.com/operator-framework/operator-controller/pull/960#discussion_r1649513635; I have been unable to figure out how to apply resources into the `cert-manager` namespace. i.e. `kustomize` seems to be locking me into a single namespace...